lib/api/api.js

```diff
@@ -21,19 +21,22 @@ import objectHead from './objectHead';
 import objectPut from './objectPut';
 import objectPutACL from './objectPutACL';
 import objectPutPart from './objectPutPart';
+import prepareRequestContexts from
+    './apiUtils/authorization/prepareRequestContexts';
 import serviceGet from './serviceGet';
 import vault from '../auth/vault';
 
-const RequestContext = policies.RequestContext;
 auth.setAuthHandler(vault);
 
 const api = {
     callApiMethod(apiMethod, request, log, callback, locationConstraint) {
+        let sourceBucket;
+        let sourceObject;
         if (apiMethod === 'objectCopy') {
             let source =
                 querystring.unescape(request.headers['x-amz-copy-source']);
-            // If client sends the source bucket/object with a leading /,
-            // remove it
+                // If client sends the source bucket/object with a leading /,
```

Comment:
Considering this applies to the code below it, why not leave the indentation as is?